README.md

```diff
@@ -93,7 +115,7 @@ config file.
 ### CLI exit codes
 
 - `0` for success (all links checked successfully or excluded/skipped as configured)
-- `1` for any unexpected runtime failures or config errors
+- `1` for missing inputs and any unexpected runtime failures or config errors
```

Comment:
Eventually we might want to have different exit codes for each case but it's out of the scope of this PR.

Reply:
Yeah, agreed